eng/Generate.ps1

```diff
@@ -3,6 +3,9 @@ param($filter, [switch]$continue, [switch]$reset, [switch]$noBuild, [switch]$fas
 
 Import-Module "$PSScriptRoot\Generation.psm1" -DisableNameChecking -Force;
 
+$timer = [System.Diagnostics.Stopwatch]::new();
```

Comment:
Maybe we can add timer in every Invoke-Typespec/Invoke-Autorest function in Generation.psm1 if we want to calculate the time of code-generation only.

Reply:
If we end up needing this granularity we can add it in the future